domains/gxlpes.json

```diff
@@ -1,11 +1,12 @@
-
-    {
-      "owner": {
-        "username": "gxlpes",
+{
+    "description": "Website portfolio",
+    "repo": "https://github.com/<gxlpes>/<gxlpes>.github.io",
```

Comment:
You don't need to add `<>`  around the repository name. That was to indicate a placeholder.
This isn't required, but would be nice.